2023/01/30 02:48:31 PM Request to telegram failed: HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot5791916657:AAF7DPD1tkco53NOlj0fLY7Jg43EHBZWLME/sendMessage?chat_id=1144450791&text=testes&parse_mode=HTML (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8686eddde0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023/01/30 02:48:33 PM Request to telegram failed: HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot5791916657:AAF7DPD1tkco53NOlj0fLY7Jg43EHBZWLME/sendMessage?chat_id=1144450791&text=testes&parse_mode=HTML (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8686d11c60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023/01/30 02:48:37 PM Request to telegram failed: HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot5791916657:AAF7DPD1tkco53NOlj0fLY7Jg43EHBZWLME/sendMessage?chat_id=1144450791&text=testes&parse_mode=HTML (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8686d10460>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023/01/30 02:48:39 PM Request to telegram failed: HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot5791916657:AAF7DPD1tkco53NOlj0fLY7Jg43EHBZWLME/sendMessage?chat_id=1144450791&text=testes&parse_mode=HTML (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8686d10e50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023/01/30 02:48:56 PM Request to telegram failed: HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot5791916657:AAF7DPD1tkco53NOlj0fLY7Jg43EHBZWLME/sendMessage?chat_id=1144450791&text=testes3&parse_mode=HTML (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8686d116c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023/01/30 02:49:00 PM Request to telegram failed: HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot5791916657:AAF7DPD1tkco53NOlj0fLY7Jg43EHBZWLME/sendMessage?chat_id=1144450791&text=testes3&parse_mode=HTML (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8686d112a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023/01/30 02:49:02 PM Request to telegram failed: HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot5791916657:AAF7DPD1tkco53NOlj0fLY7Jg43EHBZWLME/sendMessage?chat_id=1144450791&text=testes3&parse_mode=HTML (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f86a0a73760>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023/01/30 02:49:32 PM Request to telegram failed: HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot5791916657:AAF7DPD1tkco53NOlj0fLY7Jg43EHBZWLME/sendMessage?chat_id=1144450791&text=testes3&parse_mode=HTML (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8686d11ae0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023/01/30 02:49:33 PM Request to telegram failed: HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot5791916657:AAF7DPD1tkco53NOlj0fLY7Jg43EHBZWLME/sendMessage?chat_id=1144450791&text=testes3&parse_mode=HTML (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8686d10b50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2023/01/30 02:49:34 PM Request to telegram failed: HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot5791916657:AAF7DPD1tkco53NOlj0fLY7Jg43EHBZWLME/sendMessage?chat_id=1144450791&text=testes3&parse_mode=HTML (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8686d11630>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
